team/_posts/2024-04-29-john-sj-anderson.md

```diff
@@ -9,6 +9,7 @@ fediverse:
   handle: "@genehack"
 image: /images/team/john-sj-anderson.jpg
 website: https://genehack.org
+alumni: true
```

Comment:
[non-blocking comment]
I'm going to assume joined & departed tags are optional

```
joined: 2024
departed: 2025
```

Reply:
Thanks for flagging @j23414! Based on the existing use of the joined/departed tags, I assume they are only used for students/postdocs. I'll wait on @trvrb to chime in here.